### PR TITLE
Working on galaxies, solar systems, atmosphere api

### PR DESCRIPTION
**Pull request by Mooy1**

### CODE REVIEW

File: src/main/java/io/github/seggan/galactifun/api/attributes/Atmosphere.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * @author Mooy1
+ *
+ */
+@Getter
+public class Atmosphere {
+
+    public static final Atmosphere EARTH_LIKE = new Atmosphere(21, true, false, false, true, World.Environment.NORMAL);
+    public static final Atmosphere MARS_LIKE = new Atmosphere(0, false, false, false, false, World.Environment.NETHER);
+    public static final Atmosphere MOON_LIKE = new Atmosphere(0, false, false, false, false,World.Environment.THE_END);
```
Comment (Seggan) [inline src/main/java/io/github/seggan/galactifun/api/attributes/Atmosphere.java:22]: Problem with using an end environment is that itll spawn the dragon; i tried

File: src/main/java/io/github/seggan/galactifun/base/milkyway/solarsystem/Earth.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        } else {
+            PluginUtils.log(Level.WARNING, "Defaulting to world 'world'");
+            return world;
+        }
+    }
+
+    @Override
+    protected void generateChunk(@Nonnull ChunkData chunk, @Nonnull Random random, @Nonnull BiomeGrid biome, int chunkX, int chunkZ) {
+        // wont be called
+    }
```
Comment (Seggan) [inline src/main/java/io/github/seggan/galactifun/base/milkyway/solarsystem/Earth.java:59]: make `final`

File: src/main/resources/plugin.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 name: Galactifun
-author: Slimefun Addon Community
+authors: [Seggan, Mooy1] # add ur names
 description: Space addon for Slimefun
-main: io.github.seggan.galactifun.Galactifun
+main: io.github.seggan.galactifun.Galactifun # could change package name to io.github.slimefun-addon-community?
```
Comment (Seggan) [inline src/main/resources/plugin.yml:4]: i support that, except maybe `slimefunaddoncommunity`?

**Review by Seggan — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/seggan/galactifun/base/milkyway/solarsystem/Earth.java (hunk 2 above)
Comment (Mooy1) [reply src/main/java/io/github/seggan/galactifun/base/milkyway/solarsystem/Earth.java:59]: class is final

File: src/main/java/io/github/seggan/galactifun/api/attributes/Atmosphere.java (hunk 1 above)
Comment (Mooy1) [reply src/main/java/io/github/seggan/galactifun/api/attributes/Atmosphere.java:22]: rip

Comment (GallowsDove) [reply src/main/java/io/github/seggan/galactifun/api/attributes/Atmosphere.java]: can't that be blocked with a listener? We will probably want to block spawning mobs anyway

**Review by Seggan — APPROVED** [review]

**Comment by Seggan** [comment]
I'll leave the merging to you

**Pull request merged**